### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**

### CODE REVIEW

File: src/Status.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    { aces: 0, score: 0 }
+  );
+
+  if (score > 21) {
+    return "Busted...";
+  }
+
+  for (let i = 0; i <= aces; i += 1) {
+    if (score + i * 9 === 21) {
```
Comment (mikeboyle) [inline src/Status.js]: Is this saying that aces can count for 1 or 10? (1 from the initial calculation, plus 9 for each ace)? This should be 1 or 11.

File: src/Status.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const faces = new Set(["JACK", "QUEEN", "KING"]);
+
+export function Status({ cards }) {
```
Comment (mikeboyle) [inline src/Status.js]: I know this is not the standard at Codecademy but can these components all export default?

Comment (JoshuaKGoldberg) [reply src/Status.js]: Haha sure!

File: src/Status.js (hunk 1 above)
Comment (JoshuaKGoldberg) [reply src/Status.js]: Yup: the initial calculation gives them 1 point apiece, then this tries increasing that to 10 (so, adding 9) for each ace. Nice spot..

**Pull request closed without merging**